### ISSUE TITLE
Card won't show

### ISSUE THREAD

**Issue by uldirien** [nielsfaber/scheduler-card#551]
### Checklist

- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I will participate in further discussion about this issue and can help by testing (if requested).

### Card Version

2.3.3

### Component Version

3.2.8

### Bug description

the scheduler-card won't show

### Steps to reproduce

no idea

### Additional info

it looks all good, but the card just won't show

**Comment by cablesky** [comment]
I can confirm that. Nothing shows up for me either.

**Comment by SmartLiving-Rocks** [comment]
👋 Hey,  thanks for your great card!

Here is what I have experienced yesterday:

Installed a fresh instance of Home Assistant Yesterday (Home Assistant 2022.4.3) and put a copy of your scheduler-card.js in my local www folder. Added the path to Dashboard → Resources. (The local folder is accessible, as I am also using other custom cards.)

After a restart of Home Assistant, your **card** also **does not show up** for me either.
